corehq/apps/accounting/templates/accounting/partials/invoice_table.html

```diff
@@ -2,7 +2,8 @@
 {% load report_tags %}
 {% load i18n %}
 
-<a href="{{invoice_report_url}}"> {% trans "Go to Invoices for this Project Space" %}</a>
+<p class="alert alert-warning">Showing All Invoices for the Subscriber's project space</p>
+<a href="{{invoice_report_url}}"> {% trans "Go to all Invoices page for this Project Space" %}</a>
```

Comment:
Nit-picky, I know, but I'd make the capitalisation consistent with the alert above, i.e. "Go to All Invoices page for this project space".